queries/setup_dbt_profiles.sh

```
@@ -0,0 +1,36 @@
+echo "Creating directories..."
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)